project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.2
+sbt.version=1.4.5
```

Comment:
Do we really want to do this with the test issues?

Reply:
Ah... I suppose we have to if we want to bump sbt-dotty.

Reply:
Yeah, I need to finally look into the issue here